samples/data-ingestion/README.md

```diff
@@ -0,0 +1,108 @@
+---
+name: ChatGPT + Enterprise data + data ingestion
+description: Learn how to index your documents with the prepdocs script.
+languages:
+- python
+- typescript
+- bicep
+- azdeveloper
+products:
+- azure-openai
+- azure-cognitive-search
+- azure-app-service
+- azure
+page_type: sample
+urlFragment: azure-search-openai-demo-data-ingestion
+---
+# Indexing documents for the Chat App
```

Comment:
Can you put a comment that says what file this was copied/pasted from? (I think HTML comments are allowed? Or maybe there's a more Markdown-y comment)